app/models/primary_retake_english_maths.rb

```diff
@@ -1,13 +1,7 @@
 class PrimaryRetakeEnglishMaths < Base
-  attribute :retaking_english_maths, :boolean
+  attribute :retaking_english_maths, :string
 
-  validates :retaking_english_maths, inclusion: { in: [true, false], message: "You must select either yes or no" }
+  validates :retaking_english_maths, types: { method: :get_candidate_retake_gcse_status, message: "You must select either yes or no" }
 
-  def next_step
-    if retaking_english_maths == true
-      "retake_science"
-    else
-      "qualification_required"
-    end
-  end
+  OPTIONS = { yes: "222750000", no: "222750001" }.freeze
```

Comment:
May be worth putting the constants associated with the CRM into it's own module so it's not copy/pasted througout?

Reply:
Good idea